networking/src/ApiClient.ts

```diff
@@ -234,6 +237,36 @@ export class ApiClient implements IApiClient {
         this.endpoint = `https://1y2kkd6x3e.execute-api.us-east-1.amazonaws.com/${env}/createGameSession`
     }
 
+
+    async createGameSessionFromTemplate(id: string): Promise<string | null> {
+        try {
+            console.log('sup');
+            const response = await API.graphql(
+                graphqlOperation(createGameSessionFromTemplate, { input: { gameTemplateId: id } })
+            ) as { data: { createGameSessionFromTemplate: string } };
+            const result = response.data.createGameSessionFromTemplate;
+            console.log(result);
+            return result;
+        } catch (e) {
+            console.error(e);
+            return null;
+        }
+    }
```

Comment:
Have you looked into my branch for the new way to support these by creating based api clients? We should move toward that approach with these new changes to reduce the extra work in future to expand it. `ApiClient` can grow significantly a lot in length and becomes hard to maintain with adding all these new endpoints to it.